virtinst/cli.py

```diff
@@ -2568,6 +2568,7 @@ class ParserBoot(VirtCLIParser):
         "loader.secure": "loader_secure",
         "nvram.template": "nvram_template",
         "smbios.mode": "smbios_mode",
+        "initenv[0-9]*.value": "initenv[0-9]*",
```

Comment:
was this alias entry required? typically aliases should only be required when a cli option name is changed, but this is a new option

Reply:
It's not strictly necessary, no.

I just had a hard time deciding whether to use `.value` or not: `initenv[0-9]*` is shorter and mirrors the underlying XML more closely. On the other hand, `initenv[0-9]*.value` makes more sense semantically and reflects the key=value nature of environment variables better. In the end I chose to value semantics over staying close to the XML (after all, that's one of the reasons to use a CLI tool over editing the XML directly), but kept the other option as an alias as a compromise.

I don't that strongly enough about this, I just thought it was a decent compromise.

Reply:
Oh I see now, I should have looked more closely at the XML, I see what you mean.  but I'd prefer in this case to drop the 'value' part of the cli, since that term isn't present in the XML schema. So `--boot initenv0.name=FOO,initenv0=BAR`. Internally it's fine to name it initenv.value though. If everything else looks good I'll make that change before pushing

Reply:
Alright, it's fixed now.